zerver/forms.py

```diff
@@ -349,3 +349,13 @@ def clean_emails(self) -> List[str]:
             raise forms.ValidationError(_("Please enter at most 10 emails."))
 
         return emails
+
+class GoToRealmForm(forms.Form):
+    subdomain = forms.CharField(max_length=Realm.MAX_REALM_SUBDOMAIN_LENGTH, required=True)
+    next_param = forms.CharField(max_length=100, required=False)
```

Comment:
Why do we use this instead of the `?next=` pattern we use for login?

Reply:
That does make sense, and seems like a good way of solving it. 
Do you understand what our login flow is doing then? It seems different from this.